homeworks/07-mars/src/App.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { MarsPhotosService } from './services/MarsPhotosService';
+import style from './App.module.css';
+
+import { CameraList } from './components/CameraList/CameraList';
+import { Sol } from './components/Sol/Sol';
+import { ImageList } from './components/ImageList/ImageList';
+import { CameraName } from './types';
+import { RoverList } from './components/RoverList/RoverList';
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
useDispatch and useSelector at least are not used in this file, better remove unused variables and setup your IDE/eslint to help your with that